packages/dx-react-grid-material-ui/src/templates/table-group-cell/content.jsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+
+export const Content = ({
+  column, row, className, children,
+}) => (
+  <span className={className}>
```

Comment:
We should configure classNames for this component in one file. And we should combine our `className` and customer's `className`.

In your case, if a customer applies className to this component only one `className` (customer or our) would be applied.

You can see this behaviour, if you will use `contentComponent`
```jsx

const Content = props => (
  <TableGroupRow.Content {...props} className="custom-class-name" />
);
```

Same with other components.